### PR TITLE
Add 'Live' badge animation

### PR DESCRIPTION
**Pull request by raymosun**
# Info

Adds a pulsing animation to "Live" event badge.

## Changes

- [Fill in here]

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [ ] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have added new Cypress tests that are passing.

# Checklist

- [ ] I have performed a self-review of my own code.
- [ ] I have followed the style guidelines of this project.
- [ ] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [ ] My changes produce no new warnings.

# Screenshots

image

### CODE REVIEW

**Review by farisashai — COMMENTED** [review]
there's a small bug with the keyframes somewhere making it flash the border for a second at the end of the animation loop

https://github.com/acmucsd/membership-portal-ui-v2/assets/33165426/65f7af3a-9e13-4eaa-af85-cb6b1379d2ab

**Comment by SheepTester** [comment]
Screenshot_20240415-230115
Not a fan of how large the check-in card is now

**Review by alexzhang1618 — APPROVED** [review]
lgtm lgtm lgtm

File: src/styles/pages/Home.module.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
     'progress upcoming'
     'attended attended';
   grid-template-columns: 28rem 1fr;
+  grid-template-rows: auto 1fr 1fr auto;
```
Comment (alexzhang1618) [inline src/styles/pages/Home.module.scss]: WAIT REMOVE THIS FIRST

Comment (raymosun) [reply src/styles/pages/Home.module.scss]: 🫡

**Pull request merged**